ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from "@angular/core";
+import { HttpModule } from "@angular/http";
+import { Http } from "@angular/http";
+import { AutomatedUIService, Data } from "../automated-ui-service";
+
+@Component({
+  selector: "app-configurations",
+  templateUrl: "./configurations.component.html",
+  styleUrls: ["./configurations.component.css"]
+})
+export class ConfigurationsComponent {
+  public formData: Data;
+
+  exampleSchema = {
```

Comment:
this looks like a copy of the actual schema. How come you're not using schema.json?